apps/src/storage/dataBrowser/DataOverview.jsx

```diff
@@ -66,7 +66,8 @@ class DataOverview extends React.Component {
           typeof error === 'string' &&
           (error.includes('maximum number of tables') ||
             error.includes('The table name is invalid') ||
-            error.includes('The table was renamed'))
+            error.includes('The table was renamed') ||
+            error.includes('There is already a table with name'))
```

Comment:
Can you put these strings in a constants file so they live in one place?

Reply:
I don't think this would end up being a very clean solution because these error strings are parameterized, which is why we only look for a partial string match, here. So to pull them out to constants, I think we would actually need two constants for each error message- the full template string and the partial string to check for here.

Reply:
My worry is about this scenario: 
In the future, we decide to change the error messaging slightly. Someone sees the where the error is thrown and changes it directly. Then it takes a long time to figure out why the warning dialog isn't showing up anymore.

Reply:
Capturing offline conversation - I'm going to make error/warning handling throughout the data tab more robust in a follow-up PR.